## Atlas of functional connectivity relationships across rare and common genetic variants, traits, and psychiatric conditions

#### Authors: Clara A Moreau, Kumar Kuldeep, Annabelle Harvey, ...
clara.moreau@umontreal.ca

#### Last authors: Bellec Pierre, and Jacquemont Sebastien 

### Abstract
<p align="justify"> Polygenicity and pleiotropy are key genomic features of psychiatric disorders, but how they impact large-scale brain connectivity has not been investigated. 
 </p>
<p align="justify"> We analyzed resting-state fMRI data from 32,988 individuals to characterize the effects on functional connectivity of 16 multigenic copy number variants (CNVs), 10 polygenic scores, 6 cognitive and brain morphometry traits as well as 4 psychiatric conditions. </p>
<p align="justify"> Effect-sizes of CNVs on connectivity were correlated with their impact on cognition but this rapidly tapered off as CNVs increased in the number of encompassed genes. Accordingly, polygenic scores had 6-fold lower effect-sizes on connectivity compared to CNVs. Pleiotropy measured by genetic and transcriptomic correlations across 38 pairs of conditions and traits showed stablesignificant concordance with connectomic correlations observed between the same conditions and traits, suggesting a substantial causal genetic component for shared connectivity.
Despite substantial specificities, different forms of molecular risk and psychiatric conditions converged on the overconnectivity of the thalamus and somatomotor network.  </p>

#### Notebooks

[Interactive brainmaps](https://claramoreau9.github.io/Braimaps_Figure1.html) To navigate into the 19 CWAS (beta values) using nilearn</p>
[Generate Functional Connectivity profile (.md)](https://github.com/claramoreau9/NeuropsychiatricCNVs_Connectivity/blob/master/Publication_Generate_FC_alteration_mC.md) This notebook requests [stats.py](https://github.com/claramoreau9/NeuropsychiatricCNVs_Connectivity/blob/master/stats.py) and [tools.py](https://github.com/claramoreau9/NeuropsychiatricCNVs_Connectivity/blob/master/tools.py) </p>
[Get CWAS Results (.md)](https://github.com/claramoreau9/NeuropsychiatricCNVs_Connectivity/blob/master/Publication_Results_FC_alteration_mC_brainmap.md) This notebook needs [stats.py](https://github.com/claramoreau9/NeuropsychiatricCNVs_Connectivity/blob/master/stats.py) and [tools.py](https://github.com/claramoreau9/NeuropsychiatricCNVs_Connectivity/blob/master/tools.py) </p>

#### External repositories linked to this manuscript
[Permutation Tests by A Harvey](https://github.com/harveyaa/cross_cnv_paper_permutations) This repository contains all the scripts and notebooks for generating null models of betamaps (from this manuscript) through permutation and testing for significance.</p>

[Gene expression analyses by K. Kumar](https://github.com/kkumar-iitkgp-livia/GeneExp_and_CNV_FCsignatures)</p>

[Mind-GenesParallelCNV by JL Martineau](https://github.com/MartineauJeanLouis/MIND-GENESPARALLELCNV) to compute CNV calling parallel tasks in the most efficient method, in UKBiobank</p>

[Previous notebooks by C. Moreau and S. Urchs](https://github.com/surchs/Neuropsychiatric_CNV_code_supplement) </p>

#### Scripts
[to preprocess resting-state_fMRI_data with NIAK (.m)](https://github.com/claramoreau9/NeuropsychiatricCNVs_Connectivity/blob/master/to_preprocess_restingstate_fMRI_data.m) </p>
[to_generate_connectomes with NIAK using MIST64 (.m)](https://github.com/claramoreau9/NeuropsychiatricCNVs_Connectivity/blob/master/to_generate_connectomes.m) </p>
[to generate heatmaps for CellType and AHBA modules (.R)](https://github.com/claramoreau9/NeuropsychiatricCNVs_Connectivity/blob/master/Rscript_HeatMap_CellType_and_AHBAmodules.R) </p>
[to create chord diagrams (.R)](https://github.com/claramoreau9/NeuropsychiatricCNVs_Connectivity/blob/master/to_create_chord_diag.r) </p>

#### MIST ATLAS (by S.Urchs)
[Multiscale dashboard]https://simexp.github.io/multiscale_dashboard/index.html
